### PR TITLE
feat(generate-pdf): Setup nats and setting up generate-pdf service for consumption by esigns

### PR DESCRIPTION
**Pull request by fajarhikmal214**
## Overview
- feat(generate-pdf): Setup nats and setting up generate-pdf service for consumption by esigns

## Evidence:
project: SIDEBAR
title: feat(generate-pdf): Setup nats and setting up generate-pdf service for consumption by esigns
participants: @fajarhikmal214 @azophy @muhamadabduh @indraprasetya154 
screenshot: https://user-images.githubusercontent.com/79292118/223012283-fd798f65-6a96-437f-a19b-97d1f8d4068d.png

### CODE REVIEW

File: src/internal/esigns/esigns.ts [hunk 1]
```diff
@@ -1,32 +1,28 @@
...
+        const pdfGenerateUsecase = new PdfGenerateUsecase(browser)
+
+        const nats = await new Nats().connect(this.config, this.logger)
+
         const usecase = new Usecase(
             this.config,
             this.logger,
-            pdfGenerateUsecase
+            pdfGenerateUsecase,
+            nats
```
Comment (azophy) [inline src/internal/esigns/esigns.ts:22]: There is a new argument here, however I dont see any changes in `src/internal/esigns/usecase/usecase.ts` file. Did I misunderstood something about how this works?

Comment (fajarhikmal214) [reply src/internal/esigns/esigns.ts:22]: Yeah, the changes here https://github.com/jabardigitalservice/office-service-helper/pull/9 on the next PR. I just split it into 2 PRs.

**Review by indraprasetya154 — APPROVED** [review]

File: src/internal/esigns/esigns.ts (hunk 1 above)
Comment (azophy) [reply src/internal/esigns/esigns.ts:22]: I see. make sense then 👌

**Review by azophy — APPROVED** [review]

**Review by muhamadabduh — APPROVED** [review]
LGTM

**Pull request merged**